Resources/Private/.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This needs to be in the `.gitatttributes`. (Actually, I think the entry there is incorrect: `/Resources/Private/.eslintrc`)